src/components/notifier/style.css

```diff
@@ -3,18 +3,24 @@
 }
 :host[hidden] {
 	display: none;
+	
 }
 smoothly-trigger {
 	display: inline-block;
 	vertical-align: middle;
-	margin-left: 1.5em;
 }
 aside {
 	text-align: center;
 	vertical-align: middle;
-	width: 100%;
-	height: 2.8em;
-}
+	width: 100%-2em;
+	padding-left: 1em;
+	padding-right: 1em;
+	border-style: solid;
+	border-width: 1px;
+	border-color:196, 160, 0;
+	border-radius: 5px; 
+}
+
```

Comment:
We usually don't use empty lines between the blocks, so this empty line can also be removed.